nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -561,28 +561,26 @@ protected String getTableLocation(String tableName) {
     return getTableLocationPath(tableName).toString();
   }
 
-  private String metadataLocation(String tableName) {
-    return Paths.get(getTableBasePath(tableName), "metadata").toString();
-  }
-
   @SuppressWarnings(
       "RegexpSinglelineJava") // respecting this rule requires a lot more lines of code
-  private List<String> metadataFiles(String tableName) {
-    return Arrays.stream(Objects.requireNonNull(new File(metadataLocation(tableName)).listFiles()))
+  private List<String> metadataFiles(String tablePath) {
+    return Arrays.stream(
+            Objects.requireNonNull(new File((tablePath + "/" + "metadata")).listFiles()))
```

Comment:
I'd rather not mix file system APIs. This uses both Path and File. Can you get rid of `Path` and make that a string everywhere?

Reply:
I believe the original author of this file might referred this from `HiveTableBaseTest` which uses both `Path` and `File`

Reply:
@rdblue : Removed `Path` from the whole Nessie testsuite and using `File`

we can even do this for other modules like `HiveTableBaseTest` (but may be in a follow-up)

Reply:
I would probably reserve larger refactors like this to another PR, not sure they are required to avoid usage of file here?